apps/sr-frontend/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "jsx": "react-jsx",
     "target": "ESNext",
     "useDefineForClassFields": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
Why?

Reply:
because it propagates down to libs, so a lot of the `tag.src` stuff will need to be changed to `tag['src']` and I find this syntax noisy and annoying

Reply:
Tag has 'src' as a proper value

Reply:
![image](https://github.com/user-attachments/assets/9d0cca95-0be3-4726-9342-dff0de94b532)